packages/reports/src/aggregator/Aggregator.ts

```diff
@@ -0,0 +1,56 @@
+import { Aggregator as BaseAggregator } from '@tupaia/aggregator';
```

Comment:
This is largely copied from `Aggregator` in `web-config-server`. Still deciding how best to handle this duplicate code. There's a separate card for cleanup of this package where we can decide that: https://github.com/beyondessential/tupaia-backlog/issues/1283